website/docs/topics/feature-flags/feature-flag-best-practices.md

```diff
@@ -1,339 +1,188 @@
-4. **Scalability**:
+When designing a scalable feature flag system, one of the most effective strategies is to separate read and write operations into distinct APIs. This architectural decision not only allows you to scale each component independently but also provides better performance, reliability, and control.
 
-    - Scalability is a critical concern for modern applications, especially those experiencing rapid growth. Feature flagging solutions need to scale horizontally to accommodate increased workloads. Smaller payloads require fewer resources for processing, making it easier to scale your system horizontally.
+<Figure caption="Separating reading and writing of the database allows you to horizontally scale out the read APIs without scaling the write APIs." img="/img/feature-flag-horizontal-scaling.png"/>
 
-5. **Lower Infrastructure Costs**:
+By decoupling read and write operations, you gain the flexibility to scale horizontally based on the unique demands of your application. For example, if read traffic increases, you can add more servers or containers to handle the load without needing to scale the write operations. The benefits of decoupling read and write operations extend beyond just scalability; let's look at a few others:
+- **More efficient caching**: You can optimize your flag caching for read operations to reduce latency while keeping write operations consistent.
+- **Granular access control**: You can apply different security measures and access controls to the two APIs, reducing the risk of accidental or unauthorized changes.
+- **Improved monitoring and troubleshooting**: Monitoring and troubleshooting becomes more straightforward. It's easier to track and analyze the performance of each API independently. When issues arise, you can isolate the source of the problem more quickly and apply targeted fixes or optimizations.
+- **Flexibility and maintenance**: Updates to one API won't directly impact the other, reducing the risk of unintended consequences. This separation of concerns allows development teams to work on each API separately, facilitating parallel development and deployment cycles.
+- **Distributed traffic**: You can tailor load balancing strategies to the specific needs of the read and write APIs. You can distribute traffic and resources accordingly to optimize performance and ensure that neither API becomes a bottleneck under heavy load.
 
-    - When payloads are limited, the infrastructure required to support the feature flagging system can be smaller and less costly. This saves on infrastructure expenses and simplifies the management and maintenance of the system.
+## 5. Limit payload size
 
-6. **Reliability**:
+Minimizing the size of feature flag payloads is a critical aspect of maintaining the efficiency and performance of a feature flag system. The configuration of your feature flags can vary in size depending on the complexity of your targeting rules. Keeping the payload as small as possible results in:
```

Comment:
Doesn't feel like very practical advice. Check again in the original doc to see if there was more guidance there.

Reply:
More on what we mean by this here: https://docs.getunleash.io/topics/feature-flags/best-practices-using-feature-flags-at-scale#14-avoid-giant-feature-flag-targeting-lists